### ISSUE TITLE
Validation loss goes to infinity

### ISSUE THREAD

**Issue by RPalpatine** [okunator/cellseg_models.pytorch#64]
Hi okunator

Thanks for this fantastic package.

I tried playing with hovernet example notebook (ran it as is) - it seems that the validation loss goes to infinity with the first epoch.

Have you had this issue before?

**Comment by okunator** [comment]
Hi and sorry for the very late response. I think this might occur in the notebook because the `hovernet` branch uses the `SSIM`-loss. That loss can be sensitive to random bad batches leading to exploding gradients.. I'm pretty confident that if you remove that loss from the notebook, you wont bump into NaNs anymore.
